plugins/package-managers/bundler/src/main/kotlin/Bundler.kt

```diff
@@ -205,14 +205,14 @@ class Bundler(
         val scopes = mutableSetOf<Scope>()
         val issues = mutableListOf<Issue>()
 
-        val gemSpecs = resolveGemsMetadata(workingDir)
+        val gemsInfo = resolveGemsInfo(workingDir)
```

Comment:
Would it be better to name this either `gemInfo` or `gemInfos`? 

Putting the plural to `gems` sounds a bit odd to me, e.g. before like `gemsSpec`.

Reply:
I decided to do so as there is no plural of the word "information", but I somehow wanted to express that this is not just a single `GemInfo` instance.